.gitignore

```diff
@@ -8,7 +8,7 @@ yarn-error.log
 .vscode/
 .cache/
 public
-.DS_Store
+.idea/
```

Comment:
please add back `.DS_Store`

Reply:
.DS_Store is present in L5 so I removed the duplicate one